### PR TITLE
RHDEVDOCS-3868 - Review RNs for OCP 4.10.z bug fix update

### PR DESCRIPTION
**Pull request by rishumehra**
* Aligned team: Dev Tools
* For branches: Enterprise 4.10.z release notes
* Jira: [RHDEVDOCS-3868](https://issues.redhat.com/browse/RHDEVDOCS-3868)
* Direct link to doc preview: https://deploy-preview-44809--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes
* SME review: Received in the individual BZs.
* QE review: @sanketpathak
* Peer review: @Srivaralakshmi @max-cx

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc
Comment (Srivaralakshmi) [inline release_notes/ocp-4-10-release-notes.adoc:2071]:
```suggestion
* Before this update, when a secret is provided for a private repository in the *Import from Git* form, the value of the secret to fetch the private repository details is not decoded before being used. As a result, when you use the *Import from Git* form to import a private Git repository, the correct import type, and the builder image are not identified. This update resolves the issue by decoding the secret value before using it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@rishumehra PTAL. Thanks!

File: release_notes/ocp-4-10-release-notes.adoc
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Accepting the changes. Thanks @Srivaralakshmi

**Review by max-cx — APPROVED** [review]
LGTM after you review my suggested change in https://github.com/openshift/openshift-docs/pull/44809/files#r857438220

**Review by max-cx — APPROVED** [review]

File: release_notes/ocp-4-10-release-notes.adoc
Comment (opayne1) [inline release_notes/ocp-4-10-release-notes.adoc:2071]: Not sure about the use of *Import from Git* form. I am not so sure it shouldn't just be *Git import* I'll have someone verify. I made a few suggestions based on minimalism and clarity.
```suggestion
* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it is used. With his update, the secret value is now decoded before its use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[BZ#2057507])
```

**Review by opayne1 — COMMENTED** [review]
Just a few comments before we can merge! :)

**Comment by opayne1** [comment]
@rishumehra You will need to fetch/rebase your `enterprise-4.10` branch and add that bug doc text under the bug fix section in 4.10.8 z-stream entry. See https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-8

**Comment by opayne1** [comment]
LGTM!

File: release_notes/ocp-4-10-release-notes.adoc
Comment (sanketpathak) [inline release_notes/ocp-4-10-release-notes.adoc:2756]:
```suggestion
* Before this update, import strategy detection did not occur when a secret was provided for a private Git repository. Consequently, the secrets value was not decoded before it was used. With this update, the secret value is now decoded before use. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2057507[*BZ#2057507*])
```

Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Done

File: release_notes/ocp-4-10-release-notes.adoc
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: Done

**Review by RickJWagner — APPROVED** [review]
Looks good.

File: release_notes/ocp-4-10-release-notes.adoc
Comment (rishumehra) [reply release_notes/ocp-4-10-release-notes.adoc]: done

**Review by bergerhoffer — COMMENTED** [review]
LGTM too, merging per @opayne1's ping.

**Pull request merged**
